README.md

```diff
@@ -1,85 +1,59 @@
+[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
 [![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
-[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
-<!-- Add a description of module(s) in this repo -->
-TODO: Replace me with description of the module(s) in this repo
 
+This module supports creating a cloudant instance on a multi-tenant environment with standard or lite plan, or a standard plan cloudant instance with on a dedicated environment.
 
-<!-- Below content is automatically populated via pre-commit hook -->
 <!-- BEGIN OVERVIEW HOOK -->
 ## Overview
 * [terraform-ibm-cloudant](#terraform-ibm-cloudant)
 * [Examples](./examples)
-    * [Basic example](./examples/basic)
-    * [Complete example](./examples/complete)
+    * [Basic cloudant instance and database creation](./examples/basic)
+    * [Cloudant instance and database creation on dedicated host](./examples/dedicated-cloudant)
+    * [Cloudant instance, access controls, and rotation of credentials through Secret Manager](./examples/complete)
 * [Contributing](#contributing)
 <!-- END OVERVIEW HOOK -->
 
-
-<!--
-If this repo contains any reference architectures, uncomment the heading below and links to them.
-(Usually in the `/reference-architectures` directory.)
-See "Reference architecture" in Authoring Guidelines in the public documentation at
-https://terraform-ibm-modules.github.io/documentation/#/implementation-guidelines?id=reference-architecture
--->
-<!-- ## Reference architectures -->
-
-
-<!-- This heading should always match the name of the root level module (aka the repo name) -->
-## terraform-ibm-module-template
+## cloudant-module
 
 ### Usage
 
-<!--
-Add an example of the use of the module in the following code block.
-
-Use real values instead of "var.<var_name>" or other placeholder values
-unless real values don't help users know what to change.
--->
+#### Usage to create a cloudant instance on a multi-tenant environment:
 
 ```hcl
+module "cloudant" {
+  source           = "terraform-ibm-modules/terraform-ibm-cloudant/ibm"
+  version          = "X.X.X" # Replace "X.X.X" with a release version to lock into a specific release
+  source            = "git::https://github.com/terraform-ibm-modules/terraform-ibm-cloudant.git?ref=X.X.X"
+  resource_group_id = "xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX"
+  instance_name     = "my-cloudant-instance"
+}
+```
+
+#### Usage to create a cloudant instance on a dedicated environment:
 
+```hcl
+module "cloudant" {
+  source           = "terraform-ibm-modules/terraform-ibm-cloudant/ibm"
```

Comment:
this is wrong, hashicorp takes away the `terraform-ibm` so it should read: `"terraform-ibm-modules/cloudant/ibm"`